### ISSUE TITLE
heigh ho

### ISSUE THREAD

**Issue by plebhash** [plebemineira/pleblottery#1]
[heigh_ho.webm](https://github.com/plebemineira/pleblottery/assets/147345153/7f615668-56d2-4d7b-9c04-af8dbb07dafc)

cc @a-moreira @Davidson-Souza

**Comment by Davidson-Souza** [comment]
> to allow plebs to aggregate the hashrate of their multiple ASIC devices into a single solo mining "lottery ticket".

using their own node? Like we've discussed prev offline?

**Comment by plebhash** [comment]
yes, each miner deploys their own `bitcoind` as Template Provider

currently it's https://github.com/Sjors/bitcoin/tree/2023/11/sv2-poll

while we mentioned GBT on the previous discussion, I don't think RPCs are really necessary, since Sjors' fork is able to exchange SV2 messages

**Comment by plebhash** [comment]
a correction on the message above:

the initial development cycles of `pleblottery` are based on GBT

Sv2 TDP alternative is on the roadmap

**Comment by hal9000node** [comment]
Thank you for this project I am excited to learn more.

HAL
